thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/alert/content/BaseEmailContentFormatter.java

```diff
@@ -105,11 +102,11 @@
   protected Period postEventCrawlOffset;
   protected String imgPath = null;
   protected EventDataProviderManager EVENT_DATA_PROVIDER;
-  protected ThirdEyeAnomalyConfiguration THIRDEYE_CONFIG;
+  protected EmailContentFormatterConfiguration EMAIL_FORMATTER_CONFIG;
```

Comment:
please use camelCase for variables, UPPER_CASE for constants only.

Reply:
Thanks for the comment. It changes to camelCase.